src/main/kotlin/ink/abb/pogo/scraper/tasks/UpdateProfile.kt

```diff
@@ -27,14 +30,18 @@ class UpdateProfile : Task {
         val player = ctx.api.playerProfile
         val inventories = ctx.api.inventories
         try {
+            ctx.api.
```

Comment:
Oops, that line was a typo. Thankfully there is a CI system.  Removing it and resending pull request.